src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -37,7 +37,7 @@ internal static class LogMessages
     {
 #pragma warning disable 1591
         // general
-        public const string IDX10000 = "IDX10000: The parameter '{0}' cannot be a 'null' or an empty object.";
```

Comment:
>: The parameter '{0}' cannot be a 'null' or an empty object." [](start = 48, length = 61)

what is the reason that the original meaning is being removed from the string when not used? I've seen this being done before.

Reply:
We decided to remove the original meaning since it makes it more obvious that the log message is no longer being used (rather than being commented out for some other reason).